src/decorator-observable.js

```diff
@@ -9,6 +9,11 @@ export function observable(targetOrConfig: any, key: string, descriptor?: Proper
 
     // use a convention to compute the inner property name
     let innerPropertyName = `_${key}`;
+    const innerPropertyDescriptor: PropertyDescriptor = {
+      configurable: true,
+      enumerable: false,
+      writable: true,
+    };
```

Comment:
OK. Maybe they should be included or documented then? It's not obvious why this is still required (I could probably come up with at least one funny case involving `with` binding or overriden contextes).